components/desktop/freerdp/Makefile

```diff
@@ -0,0 +1,114 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Michal Nowak
+# Copyright 2020 Marco van Wieringen
+#
+
+PREFERRED_BITS=		64
+
```

Comment:
Can you convert the Makefiles into the new style, please?
BUILD_BITS=                     64
BUILD_STYLE=                 cmake
(PREFERRED_BITS=       64  can be omitted because it's the default if BUILD_BITS=64)